components/Form.js

```diff
@@ -0,0 +1,66 @@
+import { faPlus } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import styled from "styled-components";
+
+export default function Form({ name, onSubmit, placeholder, type }) {
+  return (
+    <>
```

Comment:
you don't need the fragments

Reply:
you are right... the first thing i do is to set fragments for no reason, i don't know why. Thanks for your time !